ai/Gemini_ai_call_server.py

```
@@ -0,0 +1,846 @@
+print(f"[서버] JWT_SECRET_KEY {'환경 변수에서 로드됨' if os.getenv('JWT_SECRET_KEY') else '테스트용 키 사용'}")
+
+def get_user_id_from_token(token: str) -> tuple[bool, str, str]:
+    """
+    JWT 토큰을 검증하고 사용자 ID를 반환합니다.
+    
+    Returns:
+        tuple[bool, str, str]: (성공 여부, 사용자 ID 또는 에러 메시지, 에러 타입)
+    """
+    try:
+        print(f"[서버] 토큰 검증 시작: {token[:6]}***{token[-6:] if len(token) > 12 else ''}")
+        
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)